content/blog/mlops-–-deploying-an-ml-model-in-greenlake-platform-mlops-service.md

```diff
@@ -0,0 +1,318 @@
+
+HPE GreenLake for ML Ops is an on-premises, enterprise-grade ML service, enabling developers and data scientists to rapidly build, train, and deploy ML models—from pilot to production, at any scale.
+
+This preconfigured solution comprises an optimized hardware stack and is powered by HPE Ezmeral Runtime Enterprise. It provides data scientists with self-service access to a sandbox environment for prototyping and testing, to eliminate IT provisioning delays, ensure repeatability, and accelerate time-to-value. As a fully managed solution, the HPE GreenLake for ML Ops offering frees IT from routine infrastructure management tasks.
+
+**Machine learning Lifecycle**
+
+![ML Life cycle](/img/blog_3.png "ML Life cycle")
+
+The ML project life cycle can generally be divided into three main stages: data preparation, model creation, and deployment. All three of these components are essential for creating quality models that will bring added value to your business. This process is *cyclical* because the insights gained from the existing model will help define the next model to be deployed.
+
+In this article, we focus on optimal model identified after data preparation and building model. More specifically, as an example, we shall deploy a sample ONNX model available in Kubernetes native MinIO Object storage, using Triton Inference Server.
+
+Triton Inference server supports deployment of any AI model from multiple deep learning and machine learning frameworks, including TensorRT, TensorFlow, PyTorch, ONNX, OpenVINO, Python, RAPIDS FIL, and more. Triton delivers optimized performance for many query types, including real time, batched, ensembles and audio/video streaming. To learn more about Triton Inference server, refer to the References section at the end of the post.
+
+HPE GreenLake for ML Ops platform allows customers to host their favorite cloud native applications say Mlflow, Minio, etc.
+
+**Pre-requisite**
+
+* An active service subscription to HPE GreenLake for ML Ops
+* ML Ops project has been created by ML Ops admin and user is able to launch to HPE Ezmeral Runtime Enterprise with ML Ops Project Member Role role
+* Access credentials to any S3 based object storage is available
+* Triton Inference Server container image accessible either through on-prem registry (eg Harbor) or public registry accessible.
+
+**Steps to deploy a model**
+
+1)   Validation connection to kubernetes cluster
+
+1. Click HPE GreenLake for ML Ops card in the HPE GreenLake Central **Dashboard** shows the number of previously created projects.
```

Comment:
Minor rewording:
Click the HPE GreenLake for ML Ops card in the HPE GreenLake Central **Dashboard**, which shows the number of previously created projects.